Dockerfile

```diff
@@ -1,56 +1,32 @@
-# syntax=docker/dockerfile:1.2
+FROM bitnami/minideb:bullseye AS setup
 
-# build base that sets up common dependencies of the build
-FROM rust:alpine as build-base
+RUN apt-get update && apt-get install -y \
+  curl
+  #libgcc
+  # libstdc++
 
-RUN apk add --no-cache \
-  clang clang-dev clang-libs pkgconfig bearssl-dev git \
-  gcc make g++ linux-headers protobuf protobuf-dev musl-dev
+RUN mkdir /tmp/vitalam-node/
+WORKDIR /tmp/vitalam-node/
 
-RUN set -ex; \
-  wget https://github.com/gruntwork-io/fetch/releases/download/v0.4.2/fetch_linux_amd64 -P /; \
-  mv /fetch_linux_amd64 /fetch; \
-  chmod +x /fetch; \
-  /fetch --repo="https://github.com/mozilla/sccache" --tag="~>0.2.15" --release-asset="^sccache-v[0-9.]*-x86_64-unknown-linux-musl.tar.gz$" /; \
-  tar -xvf /sccache-v*-x86_64-unknown-linux-musl.tar.gz -C /; \
-  mv /sccache-v*-x86_64-unknown-linux-musl/sccache /sccache; \
-  rm -rf /sccache-v*-x86_64-unknown-linux-musl /sccache-v*-x86_64-unknown-linux-musl.tar.gz /fetch; \
-  chmod +x /sccache;
+RUN curl -L https://github.com/gruntwork-io/fetch/releases/download/v0.4.2/fetch_linux_amd64 --output ./fetch && chmod +x ./fetch
 
-WORKDIR /build
-ARG RUST_TOOLCHAIN=nightly-2021-11-09
-RUN rustup install $RUST_TOOLCHAIN && \
-  rustup target add wasm32-unknown-unknown --toolchain $RUST_TOOLCHAIN
+RUN ./fetch --repo="https://github.com/digicatapult/vitalam-node" --tag="v2.5.1" --release-asset="vitalam-node-v2.5.1-x86_64-unknown-linux-gnu.tar.gz" ./ \
```

Comment:
Need a build-arg for the tag here. Release asset can also contain a wildcard I believe so you don't have to be specific there

Reply:
I've added the standard build arg env type situation. Please let me know if there is anything additional required.